tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -721,7 +719,7 @@ Generate table
 
 Wait until the table is generated
     [Tags]  HappyPath
-    user waits until page contains button  Generate permanent link
+    user clicks button  Share your table
```

Comment:
You'll probably want to what until the button is rendered (dev can sometimes be slow depending on the amount of people using it). So you'll want to add the below line above `user clicks button Share your table` 

`user waits until page contains button. Share your table  60`

Reply:
`user clicks button` contains `user gets button element` which in turn calls `user waits until parent contains element` - so an additional wait shouldn't be necessary.